zcash_primitives/src/transaction/components/transparent/builder.rs

```diff
@@ -9,7 +10,7 @@ use crate::{
             amount::{Amount, BalanceError, NonNegativeAmount},
             transparent::{self, Authorization, Authorized, Bundle, TxIn, TxOut},
         },
-        sighash::TransparentAuthorizingContext,
+        sighash::{SighashType, TransparentAuthorizingContext},
```

Comment:
To avoid an unused input warning, `SighashType` should be imported only if "transparent-inputs" is enabled.